studio/LonaStudio/Documents/ImageDocument.swift

```diff
@@ -42,6 +42,11 @@ class ImageDocument: NSDocument {
 
     override func read(from url: URL, ofType typeName: String) throws {
         let data = try Data(contentsOf: url, options: NSData.ReadingOptions())
-        content = NSImage(data: data)
+		if url.pathExtension == "svg" {
+			let sz = CGSize(width: 450, height: 450)
```

Comment:
We tend not to abbreviate variables in our codebase, so let's call this `size` instead